### DISCUSSION TITLE
why no windows support ?

### DISCUSSION THREAD

**Discussion by NeverBeLazyG** [gptme/gptme#179] [Q&A] [answered]
comon bro. we need windows/powershell support.

**Comment by ErikBjare** [comment, marked as answer]
I don't have a Windows machine. There's a lot of POSIX specific stuff and I don't want to learn how Windows differs.

If you want Windows support (and can't accept WSL) you can make a PR adding it, but I suspect it's not easy.

**Reply by endolith** [reply]
Can you make it obvious in the README which OSes/platforms are supported?  I just wasted time trying to install this on Windows only to get errors about not being able to install `termios` and had to research to find that that's a linux-only package.

**Reply by ErikBjare** [reply]
It is mentioned in the linked [getting started guide](https://gptme.org/docs/getting-started.html). If you want to run it on Windows you need WSL or Docker.

**Comment by endolith** [comment]
[open-interpreter](https://github.com/OpenInterpreter/open-interpreter) is similar and runs on Windows, but is unfortunately not very actively maintained (which is why I'm here looking for alternatives).

See [my spreadsheet of possible alternatives](https://docs.google.com/spreadsheets/d/15sK9oh2DRoINLfk65ooTGCDND5uTmbPsVDiCFxt2WsM/edit)

**Comment by ErikBjare** [comment]
@Kreijstal Tried vibe-porting it to Windows and suggested it wasn't difficult: https://github.com/gptme/gptme/pull/528

I don't love the idea of maintaining Windows support, but if someone drags it across the finish line I might just consider it.
